src/platform/platform_winuser.c

```diff
@@ -210,6 +210,7 @@ QuicProcessorInfoInit(
                     if (Index - ProcessorOffset < Info->Group.GroupInfo[i].ActiveProcessorCount) {
                         QuicProcessorInfo[Index].Group = i;
                         QuicProcessorInfo[Index].Index = Index - ProcessorOffset;
+                        QuicProcessorInfo[Index].MaskInGroup = 1ull << (Index - ProcessorOffset);
```

Comment:
Should we debug assert the diff is less than 64?

Reply:
Probably